libtiledbsoma/cmake/Modules/FindTileDB_EP.cmake

```diff
@@ -48,6 +48,7 @@ else()
     endif()
 
     # Try to download prebuilt artifacts unless the user specifies to build from source
+    # NB When updating the pinned URLs here, please also update in file apis/r/tools/get_tarball.R
```

Comment:
This script updates the TileDB version for C++:
https://github.com/single-cell-data/TileDB-SOMA/blob/main/scripts/update-tiledb-version.py

Let's update the script to keep R in sync (in this or another PR is fine).

Reply:
That's likely the cleanest. Marginal vote for doing it in a new and separate PR though.

Reply:
Yes, update the script in a new PR.